scenes/cslib_string.txt

```diff
@@ -314,3 +314,61 @@
 		*goto consists_of_char_list_loop
 	*set cslib_ret false
 *return
+
+
+*comment EXTRACT_TOKENS
+*comment ------------------------------
+*comment Split a given string using a separator.
+*comment The tokens are stored in a given array, up to its max.
+*comment ------------------------------
+*comment 		params:
+*comment 			p_source: the string to be split.
+*comment 			p_separator: one character to be used as a separator.
+*comment			p_array(string): the array name/reference (prefix)
+*comment 		returns:
+*comment 			(boolean): the number of tokens in the string.
+*label extract_tokens
+*params p_source p_separator p_array
+
+*set cslib_ret 0
```

Comment:
Can you move this down to where you have the lone \*return? It makes it more obvious what's being returned.
I know, *return value, would be so much nicer :(